docker/kayobe/Dockerfile

```diff
@@ -37,12 +49,6 @@ RUN dnf install epel-release -y && \
 RUN sed -i 's/# udev_rules = 1/udev_rules = 0/g' /etc/lvm/lvm.conf && \
     sed  -i 's/# udev_sync = 1/udev_sync = 0/g' /etc/lvm/lvm.conf
 
-# Workaround: rsync: --sparse-block=1024: unknown option
-# Hoepfully they have fixed this issue before this old
-# package is removed.
-# See: https://bugzilla.redhat.com/show_bug.cgi?id=2043753
-RUN dnf install -y rsync-3.1.3-14.el8 && dnf clean all
```

Comment:
I haven't tested this change with CS8, but I'm assuming I can move this to the core `dnf install`. I'm happy to move it back if I'm assuming wrongly though.